server/database.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @fileoverview database function handler
+ */
+var log = require('./log.js');
+
+/** @type {database} */
+module.exports = database;
+
+/**
+ * @constructor
+ */
+function database() {}
+
+/**
+ * Mongodb database for server.
+ * @type {!mongodb.Db}
+ */
+database.db;
+
+/**
+ * Checks the user information for signing in.
+ * @param {!mongodb.Collection} collection Database collection.
+ * @param {!mongodb.Query} query Database query.
+ * @param {!Array<!Object>} data Query result.
+ * @param {function(!Object)} cursorCallback Callback function.
+ */
+database.getOne = function(collection, query, data, cursorCallback) {
+  var cursor = collection.find(query);
+  cursor.count(function(err, count) {
+    if (err) {
+      log.serverLog(err.message);
+      return;
+    }
+    if (count > 1) {
+      log.serverLog('getting ' + count +
```

Comment:
I think serverLog takes variable number of arguments and will print the args with spaces between them. Probably this will work and be simpler:
```js
log.serverLog('getting', count, 'results while expecting one in getOne');
```